### ISSUE TITLE
maxtemp

### ISSUE THREAD

**Issue by Krullus** [jmoore52/SkywalkerRoaster#18]
Maxtemp can't handle Fahrenheit properly.

**Comment by thehack** [comment]
Yes, same for me. I couldn't figure out why my roast kept maxing out at 300F. By the time I thought to switch to Celsius, it had already been going along time. I think my total roast time was about 25 minutes, but it still came out tasting pretty nice, LOL!

**Comment by Krullus** [comment]
Should be fixed now
